webapp/channels/src/components/actions_menu/actions_menu.tsx

```diff
@@ -200,9 +199,12 @@ export class ActionMenuClass extends React.PureComponent<Props, State> {
                 key='visit-marketplace-permissions'
             >
                 <div className='visit-marketplace-text' >
-                    <FormattedMarkdownMessage
-                        id='post_info.actions.noActions'
-                        defaultMessage='No Actions currently\nconfigured for this server'
+                    <FormattedMessage
+                        id='post_info.actions.noActionsText'
+                        defaultMessage='No Actions currently{br}configured for this server'
+                        values={{
+                            br: <br/>,
+                        }}
```

Comment:
instead of adding br, can you try https://github.com/mattermost/mattermost/pull/28438#discussion_r1791547386

And see if it keeps the same UI structure?

Reply:
Followed your advice and it's working fine, The UI structure remains the same.